lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -188,11 +188,40 @@ defmodule ExUnit.Runner do
       {:error, %{test_case | state: failed}}
   end
 
-  defp run_test(config, test, context) do
+  defp run_test(true, config, test, context) do
+    run_test(:stdio, config, test, context)
+  end
+
+  defp run_test(false, config, test, context) do
+    spawn_test(config, test, context)
+  end
+
+  defp run_test(device, config, test, context) do
+    ref = make_ref()
+    try do
+      ExUnit.CaptureLog.capture_log(device, fn ->
+        send self(), {ref, spawn_test(config, test, context)}
+      end)
+    catch
+      :exit, :noproc ->
+        message =
+          "could not run test, it uses @tag :capture_log" <>
+          " but the :logger application is not running"
+        %{test | state: {:failed, {:error, RuntimeError.exception(message), []}}}
+    else
+      logged ->
+        receive do
+          {^ref, test} -> %{test | logs: logged}
+        end
+    end
+  end
+
+  defp run_test(config, %{tags: tags} = test, context) do
     EM.test_started(config.manager, test)
 
     if is_nil(test.state) do
-      test = spawn_test(config, test, Map.merge(test.tags, context))
+      capture_log? = Map.get(tags, :capture_log, false)
```

Comment:
Is there a way to enable `capture_log` by default for all tests? I didn't find anything.

I'd enable it by default in Elixir but if it's a config option I'm happy as well. E.g. something like this:

```elixir
config :logger,
  level: :debug,
  backends: [],
  capture_logs: true,
  capture_log_device: nil
```